modules/tile-converter/test/slpk-converter/slpk-converter.spec.js

```diff
@@ -0,0 +1,23 @@
+import test from 'tape-promise/tape';
+import SLPKConverter from '../../src/slpk-extractor/slpk-extractor';
+import {isBrowser} from '@loaders.gl/core';
+import {BROWSER_ERROR_MESSAGE} from '../../src/constants';
+import {cleanUpPath} from '../utils/file-utils';
+
+const SLPKUrl = 'modules/i3s/test/data/DA12_subset.slpk';
+const outputUrl = 'data';
+
+test('tile-converter - searchFromTheEnd', async (t) => {
```

Comment:
Could you rename this test as I done unified naming: https://github.com/visgl/loaders.gl/pull/2533/files.
Start from "tile-converter(slpk-extractor)#......"